### PR TITLE
[cssom][cssom-view] Update IDL tests for CSSOM and CSSOM View

### PR DESCRIPTION
**Pull request by zcorpan**
CSSOM's IDL was duplicated in interfaces/cssom.idl and
cssom/interfaces.html, and both were out of date. CSSOM View did
not have any IDL tests.

Fix CSSOM IDL test, add more objects to be tested, and add IDL
tests for CSSOM View.

Includes [Exposed=Window] per https://github.com/w3c/csswg-drafts/pull/1749.

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17701)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/267588022)

File: cssom/interfaces.html [hunk 1]
```diff
@@ -1,174 +1,85 @@
...
-<script src="/resources/testharness.js"></script>
-<script src="/resources/testharnessreport.js"></script>
-<script src="/resources/WebIDLParser.js"></script>
-<script src="/resources/idlharness.js"></script>
-<style id="styleElement">
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src=/resources/WebIDLParser.js></script>
+<script src=/resources/idlharness.js></script>
+<style id="styleElement" style="z-index: 0;">
```
Comment (rune-opera) [inline cssom/interfaces.html]: What's the z-index for?

**Review by ghost — COMMENTED** [review]
Looks generally OK. Not sure if I'm the best reviewer for this.

File: cssom/interfaces.html (hunk 1 above)
Comment (zcorpan) [reply cssom/interfaces.html]: Added a comment to clarify why it's there.

**Review by ghost — APPROVED** [review]

**Comment by tkent-google** [comment]
It seems this change broke html/dom/interfaces.html.

https://wptdashboard.appspot.com/html/dom/interfaces.html

**Comment by zcorpan** [comment]
@tkent-google ouch, sorry. I think it just needs a stub interface for `SVGElement` like it was done here. I'm on my phone now so can't PR myself at the moment.

**Pull request merged**